World/WeatherManager.gd

```diff
@@ -40,25 +51,38 @@ var wind_direction = 0 :
 		return wind_direction
 	set(new_wind_direction):
 		wind_direction = new_wind_direction
-		emit_signal("wind_direction_changed", wind_direction)
+		wind_direction_changed.emit(wind_direction)
 
 var rain_enabled := false :
 	get:
 		return rain_enabled
 	set(enabled):
 		rain_enabled = enabled 
-		emit_signal("rain_enabled_changed", enabled)
+		rain_enabled_changed.emit(enabled)
 
-var rain_density := 100.0 :
+var rain_density := 1 :
 	get:
 		return rain_density
 	set(new_rain_density):
 		rain_density = new_rain_density 
-		emit_signal("rain_density_changed", rain_density)
+		rain_density_changed.emit(rain_density)
 
-var rain_drop_size := Vector2(0.5, 0.25) :
+var rain_drop_size := 0.1 :
 	get:
 		return rain_drop_size
 	set(new_rain_drop_size):
 		rain_drop_size = new_rain_drop_size
-		emit_signal("rain_drop_size_changed", new_rain_drop_size)
+		rain_drop_size_changed.emit(new_rain_drop_size)
+
+var lightning_enabled := false :
```

Comment:
Maybe we could expose the frequency of lightnings rather than a bool? we'll probably use the Thunderstorm preset most of the time anyway, but could be practical

Reply:
I agree, i changed this with commit [33386e4](https://github.com/boku-ilen/landscapelab/pull/326/commits/33386e489d81fb0076b141e16bf2dcf09b3a31b7). A frequency of 0 automatically means lightnings are disabled.